app/services/GuidanceService.scala

```diff
@@ -112,7 +113,7 @@ class GuidanceService @Inject() (
         }
       case Right(_) =>
         logger.error(s"Referenced session ( $sessionId ) does not contain a process with processCode $processCode")
```

Comment:
I think there is a case for reducing the logging level to warn. We are not handling an internal error rather it is a case of unsupported functionality.